lucien-ios/Camera/CameraViewController.swift

```diff
@@ -35,64 +49,66 @@ class CameraViewController: UIViewController, AVCaptureVideoDataOutputSampleBuff
 
     private func startCaptureSession() {
         guard let captureDevice = captureDevice else { return }
+
         do {
             let captureDeviceInput = try AVCaptureDeviceInput(device: captureDevice)
             captureSession.addInput(captureDeviceInput)
         } catch let error as NSError {
-            let alert = UIAlertController(title: "Error", message: error.description, preferredStyle: .alert)
-            let okAction = UIAlertAction(title: "Ok", style: .cancel, handler: { [weak self] _ in
-                self?.dismiss(animated: true, completion: nil)
-            })
-            alert.addAction(okAction)
-            present(alert, animated: true, completion: nil)
+            showAlert(title: "Error", message: error.description)
         }
 
         let videoPreviewLayer = AVCaptureVideoPreviewLayer(session: captureSession)
         videoPreviewLayer.frame = cameraView.layer.bounds
-        cameraView.layer.addSublayer(videoPreviewLayer )
+        cameraView.layer.addSublayer(videoPreviewLayer)
         captureSession.startRunning()
 
-        let captureOutput = AVCaptureVideoDataOutput()
-        captureOutput.videoSettings = [(kCVPixelBufferPixelFormatTypeKey as String) : NSNumber(value: kCVPixelFormatType_32BGRA)]
-        captureOutput.alwaysDiscardsLateVideoFrames = true
-
         if captureSession.canAddOutput(captureOutput) {
             captureSession.addOutput(captureOutput)
         }
-        captureSession.commitConfiguration()
 
-        let queue = DispatchQueue(label: LucienConstants.captureQueueName)
-        captureOutput.setSampleBufferDelegate(self, queue: queue)
+        captureSession.commitConfiguration()
     }
 
+    // MARK: - IBOutlet Methods
+
     @IBAction func cameraButtonTapped(_ sender: UIButton) {
-        didTakePhoto = true
+        let settings = AVCapturePhotoSettings()
+        captureOutput.capturePhoto(with: settings, delegate: self)
     }
 
     @IBAction func cancelButtonTapped(_ sender: UIButton) {
         dismiss(animated: true, completion: nil)
     }
 
-    // MARK: - AVCaptureVideoDataOutputSampleBufferDelegate
+    // MARK: - AVCapturePhotoCaptureDelegate
+
+    func photoOutput(_ captureOutput: AVCapturePhotoOutput,
+                       didFinishProcessingPhoto photoSampleBuffer: CMSampleBuffer?,
+                       previewPhoto previewPhotoSampleBuffer: CMSampleBuffer?,
+                       resolvedSettings: AVCaptureResolvedPhotoSettings,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Vertical Parameter Alignment Violation: Function parameters should be aligned vertically if they're in multiple lines in a declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Avertical_parameter_alignment)